app/controllers/predicates/AuthorisedAction.scala

```diff
@@ -82,13 +83,14 @@ class AuthorisedAction @Inject()()(implicit val authConnector: AuthConnector,
       case enrolments ~ userConfidence if userConfidence.level >= minimumConfidenceLevel =>
         val optionalMtdItId: Option[String] = enrolmentGetIdentifierValue(EnrolmentKeys.Individual, EnrolmentIdentifiers.individualId, enrolments)
         val optionalNino: Option[String] = enrolmentGetIdentifierValue(EnrolmentKeys.nino, EnrolmentIdentifiers.nino, enrolments)
+        val optionalSessionId: String = request.headers.get("X-Session-ID").getOrElse("")
```

Comment:
Could rename `sessionId` because it's not an `Option`

Reply:
I have amended this in the next commit